### PR TITLE
Close order after successful swap

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

**Review by da-kami — APPROVED** [review]

File: cnd/src/hbit.rs [hunk 1]
```diff
@@ -2,20 +2,20 @@ use crate::{
...
 use futures::TryStreamExt;
-use std::{
-    collections::{hash_map::Entry, HashMap},
-    sync::Arc,
-};
+use std::collections::{hash_map::Entry, HashMap};
 use tokio::sync::Mutex;
 
+use crate::storage::{BtcDaiOrder, Order};
+pub use comit::{hbit::*, identity};
```
Comment (D4nte) [inline cnd/src/hbit.rs:17]:
```suggestion
use crate::storage::{BtcDaiOrder, Order};

pub use comit::{hbit::*, identity};
```

File: cnd/src/storage/db/tables/btc_dai_orders.rs [hunk 2]
```diff
@@ -3,7 +3,7 @@ use crate::storage::{
         schema::*,
         wrapper_types::{Erc20Amount, Satoshis, WeiPerSat},
     },
-    NotOpen, Order, Text,
+    NotOpen, NotSettling, Order, Text,
```
Comment (D4nte) [inline cnd/src/storage/db/tables/btc_dai_orders.rs:6]: Any reason why you defined this error outside the only file that uses it?

**Review by D4nte — APPROVED** [review]

File: cnd/src/hbit.rs (hunk 1 above)
Comment (thomaseizinger) [reply cnd/src/hbit.rs:17]: Upsi, will fix in a follow-up!

File: cnd/src/storage/db/tables/btc_dai_orders.rs (hunk 2 above)
Comment (thomaseizinger) [reply cnd/src/storage/db/tables/btc_dai_orders.rs:6]: I was thinking about that actually, for now all errors are just inside the db `errors` module but I would actually want them to be co-located yes.

I'll do that in one of my follow-ups!

**Pull request merged**